src/private/index.ts

```diff
@@ -19,4 +19,5 @@ export {
   sendCustomMessage,
   showNotification,
   uploadCustomApp
-} from "./privateAPIs";
\ No newline at end of file
+} from "./privateAPIs";
+export * from "./conversations";
```

Comment:
Please use explicit variable here export { Conversations } from "./conversations";

Reply:
Sachin both the functions here are exported